dev/doc/proof-engine.md

```diff
@@ -80,25 +78,22 @@ let cut c =
     let t = mkArrow c (Vars.lift 1 concl) in
     (** Build [X -> A]. Note the lifting of [A] due to being on the right hand
         side of the arrow. *)
-    Refine.refine { run = begin fun sigma ->
+    Refine.refine begin fun sigma ->
       (** All evars generated by this block will be added as goals *)
-      let Sigma (f, sigma, p) = Evarutil.new_evar env sigma t in
```

Comment:
OK, I just had a sudden flashback: the order of arguments is reversed, so that you need to write `(sigma, f)` instead, and similarly below.

Reply:
Fixed [I think]